switchsai/sailag.c

```diff
@@ -271,6 +282,16 @@ static sai_status_t sai_create_lag_member(
 
   api_lag_member_info.port_id = attribute->value.u32;
 
+  attribute = get_attr_from_list(SAI_LAG_MEMBER_ATTR_EGRESS_DISABLE, attr_list,
+                                 attr_count);
+  if (attribute == NULL) {
+    status = SAI_STATUS_INVALID_PARAMETER;
```

Comment:
Only attributes that are _MANDATORY_ON_CREATE_ should return _SAI_STATUS_MANDATORY_ATTRIBUTE_MISSING_ in case of failure. Since, these are not mandatory attributes, I don't think that's a valid return type.